requirements_base.txt

```diff
@@ -7,4 +7,4 @@ pooch>=1.5
 decorator
 packaging
 jinja2
-importlib_resources>=5.10.2
+importlib_resources>=5.10.2; python_version < '3.9'
```

Comment:
Might need to be 3.10, or matplotlib has a bug? Not sure

https://github.com/matplotlib/matplotlib/blob/5b1e02d734245a8f39c1c057f04539ab1c94738c/setup.py#L337

Reply:
It depends on which functionality you need. The version in the standard library lags behind the PyPI package: https://pypi.org/project/importlib-resources/